site/docs/spark-queries.md

```diff
@@ -219,9 +221,33 @@ SELECT * FROM prod.db.table.files
 +-------------------------------------------------------------------------+-------------+--------------+--------------------+--------------------+------------------+-------------------+------------------+-----------------+-----------------+--------------+---------------+
 ```
 
+### All Data Files
+
+To show a table's valid data files and each file's metadata, run:
```

Comment:
I don't think it is a good idea to end a paragraph with "run:" and then put two callouts between it and the SQL that it is referencing.